druid/src/core.rs

```diff
@@ -446,7 +446,9 @@ impl<T: Data, W: Widget<T>> WidgetPod<T, W> {
                 Target::Window(_) => Event::Command(cmd.clone()),
                 Target::Widget(id) if *id == child_ctx.widget_id() => Event::Command(cmd.clone()),
                 Target::Widget(id) => {
-                    recurse = child_ctx.base_state.children.contains(id);
+                    // Recurse when the target widget could be our descendant.
+                    // The bloom filter we're checking can return false positives.
```

Comment:
> The bloom filter we're checking can return false positives.

I think the renaming to `may_contain` should make it clear enough, as this comment has to be applied to too many places to stay consistent (and those were definitly to many `to`s 😅 ).

Reply:
Just the name `may_contain` will make it clear enough to someone who already knows the function or will go read its docs. Others who are just glancing at the code at the call site won't necessarily know.

I also don't think the three places this comment was added to is too many. Once we're up to 10 or something we can reconsider.